### PR TITLE
Added support for Turkish language in the web interface.

### PR DESCRIPTION
**Pull request by hasansezertasan**
This will help to expand the user base and make the project more accessible to a wider audience.

### CODE REVIEW

File: starlette_admin/statics/i18n/dt/tr.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+				"not": "Değildir",
+				"notBetween": "Arasında Değil",
+				"notEmpty": "Boş Değil"
+			},
+			"number": {
+				"between": "Arasında",
+				"empty": "Boş",
+				"equals": "Eşittir",
+				"gt": "Büyüktür",
+				"gte": "Büyük Eşit",
```
Comment (alperiox) [inline starlette_admin/statics/i18n/dt/tr.json:107]: buyuk esittir demek daha dogru olmaz mi? hemen asagida kucuk esittir denmis.

File: starlette_admin/statics/i18n/dt/tr.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+				"lt": "Küçüktür",
+				"lte": "Küçük Eşittir",
+				"not": "Değildir",
+				"notBetween": "Arasında Değil",
+				"notEmpty": "Boş Değil"
+			},
+			"string": {
+				"contains": "Koşul",
```
Comment (alperiox) [inline starlette_admin/statics/i18n/dt/tr.json:115]: icerige bagli olarak uygun bir ceviri olabilir, fakat icerige hakim olmadigimdan cevirinin dogru olup olmadigindan emin degilim.

File: starlette_admin/statics/i18n/dt/tr.json (hunk 1 above)
Comment (hasansezertasan) [reply starlette_admin/statics/i18n/dt/tr.json:107]: Review için teşekkürler. Haklısın, düzeltiyorum.

File: starlette_admin/statics/i18n/dt/tr.json (hunk 2 above)
Comment (hasansezertasan) [reply starlette_admin/statics/i18n/dt/tr.json:115]: Yorum için teşekkürler. Haklısın, düzeltiyorum.

**Review by alperiox — APPROVED** [review]
I didn't find a problem or a mistranslation, thanks for your work!

Bir problem ya da yanlis ceviri goremedim, eline saglik!

**Comment by hasansezertasan** [comment]
Thank you @alperiox for the review. I'm gratefull! 🙏

Gözden geçirdiğin için teşekkürler @alperiox. Minnettarım! 🙏

**Review by jowilf — APPROVED** [review]

**Comment by jowilf** [comment]
Thanks @hasansezertasan & @alperiox for your contributions 🎉

**Comment by hasansezertasan** [comment]
@jowilf Thank you for bringing this project to life, it inspired me so much and with this journey of translation, I improved my toolset with a bunch of great tools. I'll be watching it closely and hoping to contribute more.

**Comment by jowilf** [comment]
Thank you for the feedback! I'm really glad this project has inspired you and helped you grow your skills. Your contributions are welcome

**Pull request merged**